ssl/s3_lib.c

```diff
@@ -3734,7 +3734,7 @@ long ssl3_ctrl(SSL *s, int cmd, long larg, void *parg)
             if (SSL_CONNECTION_IS_TLS13(sc) && sc->s3.did_kex)
                 id = sc->s3.group_id;
             else
-                id = sc->session->kex_group;
+                id = (sc->session != NULL) ? sc->session->kex_group : NID_undef;
```

Comment:
Nit: I would remove the extra parentheses.

Reply:
> Nit: I would remove the extra parentheses.

I wouldn't :-)

Reply:
So could you both please approve? :)